Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt

```diff
@@ -1,51 +1,51 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//釣り=
-//ツリーイング=
-//カヤック=
-//シュノーケリング=
-//トレッキング=
-//ホエールウォッチング=
+釣り=Fishing
+ツリーイング=Treeing
+カヤック=Kayaking
+シュノーケリング=Snorkeling
+トレッキング=Trekking
+ホエールウォッチング=Whale Watching
 バナナボート=Banana Boat
-//水牛=
-//ネコ=
-//ヤシガニ=
-//鳥の雛=
-//ヤマネコ=
-//マングローブ=
-//アサガオ=
-//パワースポット=
+水牛=Water Buffalo
+ネコ=Cats
+ヤシガニ=Coconut Crab
+鳥の雛=Baby Birds
+ヤマネコ=Wildcats
+マングローブ=Mangrove
+アサガオ=Morning Glory
+パワースポット=Power Spots
 ハイビスカス=Hibiscus
-//ガジュマル=
-//月桃=
-//流れ星=
-//星空=
-//サガリバナ=
-//シオマネキ=
-//サンゴ=
-//クマノミ=
-//チンアナゴ=
-//マンタ=
-//ウミガメ=
-//海ほたる=
-//星の砂=
-//お土産=
-//名物料理=
-//シークヮーサージュース=
-//ぬいぐるみ=
-//ウムイの鐘=
-//紅型=
-//三線=
-//シーサー=
-//キスの味=
-//初経験=
-//ぬくもり=
-//体の相性=
-//甘い夜=
-//胸の話=
-//お尻の話=
-//性器=
-//フェチ=
-//好きな体位=
-//欲求不満=
+ガジュマル=Banyan Trees
+月桃=Shell Ginger
+流れ星=Shooting Stars
+星空=Night Sky
+サガリバナ=Barringtonia
+シオマネキ=Fiddler Crab
+サンゴ=Coral
+クマノミ=Clownfish
+チンアナゴ=Spotted Garden Eel
+マンタ=Manta Ray
+ウミガメ=Sea Turtles
+海ほたる=Sea Fireflies
+星の砂=Star Sand
+お土産=Souvenirs
+名物料理=Specialty Dishes
+シークヮーサージュース=Shikuwasa juice
+ぬいぐるみ=Stuffed Animals
+ウムイの鐘=Umui Bells
+紅型=Bingata
+三線=Shamisen
+シーサー=Shisa
+キスの味=Taste of Kisses
+初経験=First Experience
+ぬくもり=Warmth
+体の相性=Body Chemistry
+甘い夜=Sweet Nights
+胸の話=Breast Stuff
+お尻の話=Butt Stuff
+性器=Genitals
+フェチ=Fetishes
+好きな体位=Favorite Position
+欲求不満=Frustration
```

Comment:
Sexual frustration (desire)

Reply:
They're color coded as well in game, so it's obvious what it is.  In context it seems much more frustration than desire.